pypoolstation/__init__.py

```diff
@@ -206,23 +212,40 @@ async def sync_info(self):
             pass
       
         try:
-            self.uv = info["vars"][API_SIGNS["uv"]] == "0"
+            self.uv_available = info["vars"][API_SIGNS["uv_available"]] != "-"
+
+            # New Logic based on 'bu' (uv_ballast). This probably only works for non-prioriatary UV
+            # lights to detect the light state.
+            bu_val = info["vars"].get(API_SIGNS["uv_ballast"])
+            if bu_val == "-":
```

Comment:
I take that "-" means something like "auto", meaning the internal schedule of the chlorinator will be in charge, right?